src/components/StatusLight/StatusLight.test.tsx

```diff
@@ -0,0 +1,66 @@
+import { render } from '../../../test/testUtils';
+import StatusLight, { StatusLightProps } from './StatusLight';
+import { AVAILABLE_LIGHT_COLORS, COLOR_STYLES_MAP } from './StatusLightContants';
+
+const defaultProps = {
+  color: AVAILABLE_LIGHT_COLORS.GREEN,
+  message: 'your entity is available',
+  pulse: false,
+};
+const setup = (props: Partial<StatusLightProps>) => {
+  const combinedProps = { ...defaultProps, ...props };
+
+  return render(<StatusLight {...combinedProps} />);
+};
+
+describe('The StatusLight component', () => {
+  describe('Adjusting the lights color', () => {
+    it('can display a green light', () => {
+      const { container } = setup({ color: StatusLight.colors.GREEN });
+
+      expect(container.querySelector('circle')).toHaveClass(
+        COLOR_STYLES_MAP[StatusLight.colors.GREEN],
+      );
+    });
+
+    it('can display a yellow light', () => {
+      const { container } = setup({ color: StatusLight.colors.YELLOW });
+
+      expect(container.querySelector('circle')).toHaveClass(
+        COLOR_STYLES_MAP[StatusLight.colors.YELLOW],
+      );
+    });
+
+    it('can display a red light', () => {
+      const { container } = setup({ color: StatusLight.colors.RED });
+
+      expect(container.querySelector('circle')).toHaveClass(
+        COLOR_STYLES_MAP[StatusLight.colors.RED],
+      );
+    });
+  });
+
+  describe('When displaying a message', () => {
+    it('displays the given message text', () => {
+      const { getByText } = setup({ message: 'Your entity is ready' });
```

Comment:
Can you update the tests to use the more modern syntax, please? Example...

```
setup();
```
or
```
const { user } = setup();
```

and then

```
screen.getBy____
```

Reply:
Ah ok, I just copied stuff across from the UI! But good point.